shared/video-sdk/develop/real-time-transcription/project-implementation/electron.mdx

```diff
@@ -2,28 +2,28 @@
 
 ### Implement the user interface
 
-To create a simple interface for testing Real-Time Transcription functionality, add the following buttons to the UI.
+To create a simple interface to test this feature, add the following buttons to the UI.
 
 1. Open `index.html` and add the following lines after `<button type="button" id="leave">Leave</button>`:
 
     ```html
-    <button type="button" id="speechToText">Start speech-to-text</button>
-    <button type="button" id="speechToTextQuery">Query speech-to-text</button>
+    <button type="button" id="realTimeTranscription">Start real-time transcription</button>
+    <button type="button" id="realTimeTranscriptionQuery">Query real-time transcription</button>
     ```
 
 ### Handle the system logic
 
-To integrate HTTP and JSON capabilities into your <Vpl k="CLIENT" /> for speech-to-text:
+To integrate HTTP and JSON capabilities into your <Vpl k="CLIENT" /> for Real-Time Transcription:
```

Comment:
`for **r**eal-**t**ime **t**ranscription.`
I suppose no need to change it to upper case , as the same is in lower case in original doc and we are following the the same case as it is in original doc for rest all places in the doc.